packages/core/src/layout/ItemCard/ItemCard.tsx

```diff
@@ -64,13 +71,22 @@ export const ItemCard = ({
         {tags?.map((tag, i) => (
           <Chip label={tag} key={`tag-${i}`} />
         ))}
-        <Typography variant="body2" paragraph className={classes.description}>
+        <Typography
+          variant="body2"
+          paragraph
+          className={`${classes.description} ${
+            tags && tags.length > 0 ? classes.withTags : ''
+          }`}
+        >
           {description}
         </Typography>
         <div className={classes.footer}>
-          <Button onClick={onClick} color="primary">
-            {label}
-          </Button>
+          {!href && (
+            <Button onClick={onClick} color="primary">
+              {label}
+            </Button>
+          )}
+          {href && <Link to={href}>{label}</Link>}
```

Comment:
This looks kind of strange when you hover because the button text is underlined. When you don't hover it looks the same as a raw `<Link />`, maybe it has some more padding.